Public/Src/Engine/Dll/Engine.cs

```diff
@@ -1873,9 +1873,16 @@ void CheckArtifactFolersAndEmitNoIndexWarning(params AbsolutePath[] paths)
 
                                     WorkerService workerservice = null;
 
-                                    if (Configuration.Distribution.BuildRole == DistributedBuildRoles.Worker)
+                                    if (Configuration.Distribution.BuildRole != DistributedBuildRoles.None)
                                     {
-                                        workerservice = m_workerService;
+                                        if (Configuration.Distribution.BuildRole == DistributedBuildRoles.Master)
+                                        {
+                                            m_masterService.EnableDistribution(engineSchedule);
+                                        }
+                                        else if (Configuration.Distribution.BuildRole == DistributedBuildRoles.Worker)
+                                        {
+                                            workerservice = m_workerService;

```

Comment:
> //C [](start = 15, length = 4)

Please uncomment this code.

Reply:
I have un-commentated the line in new commit.